src/drive/containers/File.jsx

```diff
@@ -196,9 +196,7 @@ const FileAction = ({ onClick }) => (
       styles['fil-content-file-action']
     )}
   >
-    <Button theme="action" onClick={onClick} extension="narrow">
-      <Icon icon="dots" color="charcoalGrey" width="17" height="17" />
-    </Button>
+    <Button theme="action" onClick={onClick} extension="narrow" icon="dots" />
```

Comment:
I guess you should use `icon={<Icon icon="dots" color="charcoalGrey" width="17" height="17" />}` here too.